### ISSUE TITLE
comparisons with related work

### ISSUE THREAD

**Issue by tdhock** [tdhock/atime#40]
https://github.com/airspeed-velocity/asv
https://github.com/benchmark-action/github-action-benchmark
https://github.com/conbench/conbench
https://github.com/lorenzwalthert/touchstone
@DorisAmoakohene

**Comment by tdhock** [comment]
probably the main difference is that atime varies N so we get a whole curve, whereas others do not

**Comment by tdhock** [comment]
@DorisAmoakohene can you add these packages to the readme please? https://github.com/tdhock/atime?tab=readme-ov-file#related-work only mentions testComplexity and bench packages

**Comment by DorisAmoakohene** [comment]
https://github.com/DorisAmoakohene/PerformanceTest_data.table/tree/master

solved

**Comment by tdhock** [comment]
please send atime a PR

**Comment by DorisAmoakohene** [comment]
Alright

**Comment by tdhock** [comment]
@DorisAmoakohene  can you please translate your Table 1 from overleaf/latex to markdown and put it on the readme under related work?

**Comment by DorisAmoakohene** [comment]
https://github.com/tdhock/atime/blob/main/README.org
@tdhock see this if its good

**Comment by tdhock** [comment]
yes that is good but even better would be if you could add some links
* under first column to the github or docs page of the software
* under Github actions column, link to an example web page of results

**Comment by DorisAmoakohene** [comment]
https://github.com/tdhock/atime/blob/main/README.org#github-action-for-continuous-performance-testing

done

**Comment by tdhock** [comment]
looks good thanks
